server/util/db/db.go

```diff
@@ -140,88 +210,60 @@ func parseDatasource(datasource string) (string, string, error) {
 	return "", "", fmt.Errorf("No database configured -- please specify at least one in the config")
 }
 
+// sqlLogger is a GORM logger wrapper that supresses "record not found" errors.
 type sqlLogger struct {
-	logQueries bool
-	delegate   gorm.Logger
+	logLevel logger.LogLevel
+	delegate logger.Interface
 }
```

Comment:
Another option here that might save you some lines is making delegate an anonymous field: (http://golangtutorials.blogspot.com/2011/06/anonymous-fields-in-structs-like-object.html)

I think this would let you remove the Info, Warn, and Error methods you've had to define that just call the delegate. You could still of course keep logLevel as a named member.

Reply:
Good idea, thanks for the pointer